package.json

```diff
@@ -2,6 +2,61 @@
+    "check-errors": "yarn lint --fix && tsc --noEmit",
+    "cypress:ct": "cross-env start-server-and-test 'yarn start' http-get://localhost:3000 'yarn cypress:run'",
+    "cypress:run": "cross-env MOCHAWESOME_HTML='false' MOCHAWESOME_OVERWRITE='false' cypress run -r mochawesome -s 'tests/cypress/integration/smoke/*'",
+    "eject": "react-scripts eject",
+    "integration-test": "start-server-and-test 'serve build -l 3000' http://localhost:3000 'cypress run'",
+    "ipfs-build": "cross-env PUBLIC_URL=\".\" GENERATE_SOURCEMAP=false react-app-rewired build",
+    "lint": "eslint \"./src/**/*.{ts,tsx,js}\"",
+    "report:generate": "marge index.json --reportDir public-report --assetsDir public-report/assets --reportPageTitle index.html",
+    "report:merge": "mochawesome-merge mochawesome-report/*.json > index.json",
+    "serve": "serve -s ./build",
+    "start": "yarn && react-scripts start",
+    "synpress:ct": "cross-env start-server-and-test 'yarn start' http-get://localhost:3000 'yarn synpress:run'",
+    "synpress:open": "cross-env MOCHAWESOME_HTML='false' MOCHAWESOME_OVERWRITE='false' NETWORK_NAME='rinkeby' CYPRESS_SKIP_RESOURCES_WAIT='true' FAIL_ON_ERROR='0' PRIVATE_KEY='367f6baeb27b33408c85040e13d1ba3e2a173a9112eb3b103f79df0729ddda72' ETHERSCAN_KEY='25VF97IQZ8EFIYSYFI72UCTJJII3HSZU7Z' synpress open --configFile tests/synpress/synpress.json",
+    "synpress:run": "cross-env MOCHAWESOME_HTML='false' MOCHAWESOME_OVERWRITE='false' NETWORK_NAME='rinkeby' CYPRESS_SKIP_RESOURCES_WAIT='true' FAIL_ON_ERROR='0' PRIVATE_KEY='367f6baeb27b33408c85040e13d1ba3e2a173a9112eb3b103f79df0729ddda72' ETHERSCAN_KEY='25VF97IQZ8EFIYSYFI72UCTJJII3HSZU7Z' synpress run --configFile tests/synpress/synpress.json -r mochawesome --reporterOptions 'overwrite=false, html=false, json=true'",
+    "test": "react-scripts test --env=jsdom"
+  },
+  "resolutions": {
+    "@typescript-eslint/eslint-plugin": "^5.20.0",
+    "@typescript-eslint/parser": "^5.20.0",
+    "ansi-html": "^0.0.8",
+    "ansi-regex": "^5.0.1",
+    "axios": "^0.21.2",
+    "browserslist": "^4.16.5",
+    "dns-packet": "^1.3.2",
+    "glob-parent": "^5.1.2",
+    "hosted-git-info": "^2.8.9",
+    "immer": "^9.0.6",
+    "is-svg": "^4.3.0",
+    "minimist": "^1.2.6",
+    "node-fetch": "^2.6.7",
+    "node-forge": "^1.3.0",
+    "path-parse": "^1.0.7",
+    "postcss": "^7.0.36",
+    "postcss-safe-parser": "^4.0.2",
+    "simple-get": "^2.8.2",
+    "ssri": "^7.1.1",
+    "tmpl": "^1.0.5",
+    "trim": "^0.0.3",
+    "underscore": "^1.12.1",
+    "url-parse": "^1.5.9",
+    "ws": "^7.4.6"
+  },
+  "dependencies": {
+    "@web3-react/walletlink-connector": "^6.2.8",
+    "aos": "^2.3.4",
+    "arb-ts": "^1.0.2",
+    "multiformats": "^9.4.10",
+    "query-string": "^6.13.8",
+    "rc-texty": "^0.2.0",
+    "react-fast-marquee": "^1.3.1",
+    "react-use": "^15.3.3"
+  },
   "devDependencies": {
     "@apollo/client": "^3.4.7",
     "@ethersproject/experimental": "^5.4.0",
```

Comment:
It's best to organize these packages here. Most of these packages belong to `dependencies`.

Reply:
Ok I will move all devDependencies and dependencies separate.